packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -32,7 +32,7 @@ for (const packageName of names) {
           try {
             loadChannel.publish({ name, version, file })
 
-            moduleExports = hook(moduleExports)
+            moduleExports = hook(moduleExports, version)
```

Comment:
unrelated change. I think this comes in really handy, but if it can have issues I can separate in a different PR